server/service/src/system_user.rs

```diff
@@ -0,0 +1,20 @@
+use repository::{RepositoryError, StorageConnection, UserAccountRow, UserAccountRowRepository};
+use util::uuid::uuid;
+
+pub fn system_user(connection: &StorageConnection) -> Result<UserAccountRow, RepositoryError> {
+    let system_user = UserAccountRow {
+        id: uuid(),
```

Comment:
I wonder if this might be better as a static id - generating a new UUID on every access will require searching for the `System` username: it isn't possible to search for the system user by ID now

Reply:
I had the same thought... but the Static ID gave a 'NOT UNIQUE' error when trying to transfer requisitions 🤔

Reply:
We don't need to insert the row at all, sorry I should have been more specific, what i was meaning, similar to special name:
* Make a new constant here: https://github.com/openmsupply/open-msupply/blob/develop/server/util/src/constants.rs, something like 'SYSTEM_USER_ID =  "omsupply_system_user`
* Use that constant in processor logs, ooh, we can make a new 'system user context method' and use here: https://github.com/openmsupply/open-msupply/blob/7043d27cb03796af38dfbc5a7ab57464ddebec92/server/service/src/processors/transfer/requisition/mod.rs#L68
* Whenever we query for user we should intercept user id with SYSTEM_USER_ID and return the system user, i thought this should be done in `service` layer but just realised that we don't yet use service layer in user loader : /. So for now, although it's a little bit hacky, add that user to the loader result if `keys` includes SYSTEM_USER_ID.

Reply:
[8563099](https://github.com/openmsupply/open-msupply/pull/666/commits/8563099e0beb5c369b20e12dd9e36bef7fb9f478)